src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -136,7 +132,34 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         }
     }
 
-    class GrainIdConverter : JsonConverter
+    public class IPEndPointConverter : JsonConverter
```

Comment:
Nit: Why is IPEndPointConverter `public` visibility, but others are `internal` ?